benchmark-canisters/src/memory_manager.rs

```diff
@@ -56,3 +57,21 @@ pub fn memory_manager_overhead() -> BenchResult {
         }
     })
 }
+
+/// Benchmarks the `MemoryManager` by allocating a large number of buckets.
+#[ic_cdk_macros::query]
+pub fn memory_manager_buckets_allocation() -> BenchResult {
```

Comment:
A couple of points:

1. I don't understand what the intent is of this benchmark. What more does it add compares to the ones that we already have?
2. This is totally unrelated to adding benchmarks to CI, so let's please move it out of this PR.

Reply:
If the goal is to understand the performance regression with the memory.free changes, then we need a scenario where we declare tens of thousands of buckets, and then we benchmark how long it takes to call `memory.grow`. But, again, that's a different PR.